doc/classes/OS.xml

```diff
@@ -4,7 +4,7 @@
 		Operating System functions.
 	</brief_description>
 	<description>
-		Operating System functions. OS wraps the most common functionality to communicate with the host operating system, such as the clipboard, video driver, date and time, timers, environment variables, execution of binaries, command line, etc.
+		Operating System functions. OS wraps the most common functionality to communicate with the host operating system, such as the clipboard, video driver, timers, environment variables, execution of binaries, command line, etc.
```

Comment:
I guess `timers` is left to reference `delay_*` methods? I'm not sure about it, "timer" is usually a non-blocking thing, so it might be better to change it to "…, delays, …"?

Reply:
sure thing, I wasn't certain what to do there either